src/network/networkd-link.c

```diff
@@ -2618,6 +2618,21 @@ static int link_set_ipv6_mtu(Link *link) {
         return 0;
 }
 
+static int link_is_static_address_configured(Link *link, Address *address) {
+        assert(link);
+        assert(address);
+
+        if (link->network) {
```

Comment:
we generally try to keep the indentation levels low, and hence prefer early exit over deep nesting of if blocks. i.e. in this case just do 
```c
if (!link->network)
        return false;
```